<launch>

	<!-- motor_state_topic: The topic containing motor state information
		 servo_state_topic: The topic containing servo state information -->
	<node pkg="lab4" name="motion_model" type="MotionModel.py" output="screen" >
	  <param name="motor_state_topic" type="string" value="/vesc/sensors/core"/>
	  <param name="servo_state_topic" type="string" value="/vesc/sensors/servo_position_command"/>
	</node>

</launch>
